src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -713,6 +713,17 @@ public function simplePaginate($perPage = null, $columns = ['*'], $pageName = 'p
     }
 
     /**
+     * Create and return and un-saved model instance.
+     *
+     * @param  array  $attributes
+     * @return \Illuminate\Database\Eloquent\Model
+     */
+    public function make(array $attributes = [])
```

Comment:
It works when called from the Model context because `make()` (and `create()` which is also not static) are non-existent on Model.php, triggering `__callStatic()` which then does:
`return (new static)->$method(...$parameters);` calling the dynamic method on Eloquent Builder dynamically. This appears to be indended functionality based on [this comment](https://github.com/laravel/framework/blob/5.4/src/Illuminate/Database/Eloquent/Model.php#L1336)

At least that's how it seems, but I'm kind of a source-diving n00b so I could be reading this wrong.